packages/frontend/src/components/Game/GameBoard/TileGrid/index.test.jsx

```diff
@@ -5,8 +5,8 @@ import { mockPhotos } from '__mocks__/api/mockPhotos';
 
 describe('TileGrid component', () => {
   it('renders all tiles', () => {
-    const tiles = createTilesFromPhotos(mockPhotos, true)
-    
+    const tiles = createTilesFromPhotos(mockPhotos, true);
```

Comment:
I believe the second argument of the `createTilesFromPhotos` function is an options object which would have `shuffle: true` as a property

Reply:
It could also be set as the default value in the function definition so it doesn't have to explicitly be passed unless we don't want the tiles shuffled